### PR TITLE
add get self claims docs

### PR DESCRIPTION
**Pull request by gmanningcr**
connects CorporateRewards/MyRewards#7650

### CODE REVIEW

File: source/includes/api/performance/_self_claims.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+date_of_sale_start_date | `string` | A parameter for scoping claims with a date_of_sale from the given date onwards. Optional.
+date_of_sale_end_date | `string` | A parameter for scoping claims with a date_of_sale on or before the given date. Optional.
+page | `integer` | The requested page number. Defaults to page 1 if not supplied. Optional.
+
+
+#### Response Attributes
+
+Attribute | Type | Info
+--------- | ---- | ----
+id | integer | the id of the claim
```
Comment (jcleary) [inline source/includes/api/performance/_self_claims.md:166]: do you need to backtick the types here?

File: source/includes/api/performance/_self_claims.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
+## Self Claims
+
+### Get Self Claims
+
+Endpoint to get self claims against a specific self claim promotion. Claims can be scoped down via passed in parameters. If no url parameters are provided then all claims against the requested promotion are returned. Results are paginated at 100 records per page.
```
Comment (rjpaskin) [inline source/includes/api/performance/_self_claims.md:5]:
```suggestion
Endpoint to get self claims against a specific self-claim promotion. Claims can be scoped down via passed-in parameters. If no URL parameters are provided then all claims against the requested promotion are returned. Results are paginated at 100 records per page.
```

File: source/includes/api/performance/_self_claims.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+            "id": 4099,
+            "sale_date": "29/01/2019",
+            "product_or_activity_ref": "p1",
+            "quantity": 150,
+            "points_value": 150,
+            "user_id": 505,
+            "status": "pending",
+            "declined_reason": null,
+            "created_at": "30/01/2019",
+            "updated_at": "07/01/2021",
```
Comment (rjpaskin) [inline source/includes/api/performance/_self_claims.md:38]: ISO format for dates? Don't know what we use elsewhere in the API

**Review by rjpaskin — CHANGES_REQUESTED** [review]
Use "filter" instead of "scope"?

File: source/includes/api/performance/_self_claims.md (hunk 3 above)
Comment (gmanningcr) [reply source/includes/api/performance/_self_claims.md:38]: I replicated the allocated claims endpoint format here - am checking wtih Jules to see what she wants

**Review by jcleary — APPROVED** [review]

**Review by rjpaskin — APPROVED** [review]

**Pull request merged**